common/static/common/templates/discussion/search-alert.underscore

```diff
@@ -4,6 +4,7 @@
     </div>
 
     <div class="search-alert-controls">
-      <a href="#" class="dismiss control control-dismiss"><i class="icon fa fa-remove"></i></a>
+      <a href="#" class="dismiss control control-dismiss"><span class="icon fa fa-remove" aria-hidden="true"></span>
+      <span class="sr"><%= gettext("Close") %></span></a>
```

Comment:
@ssemenova This should be <%- gettext("Close") %> so that the text is HTML-escaped. Even though the English string has no characters that need to be escaped, we can't control the translations, so we just always escape to be safe.